src/main/java/bowling/Player.java

```diff
@@ -0,0 +1,44 @@
+package bowling;
+
+import java.util.Objects;
+import java.util.regex.Pattern;
+
+public class Player {
+
+    private static final String NAMING_REGEX = "^[a-zA-Z]{3}$";
+    private final String name;
+
+    private Player(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    private static void validate(String name) throws IllegalArgumentException {
+        if (!Pattern.matches(NAMING_REGEX, name)) {
```

Comment:
미션 진행과 큰 상관은 없지만, 작은 팁 하나 드릴게요 :)
계속해서 똑같은 정규식 패턴을 사용한다면, 정규식을 `String` 상수로 사용하는 것보다, compile 된 `Pattern` 상수로 사용하는 편이 낫습니다. Pattern을 컴파일 하는데 리소스가 은근 잡아먹기 때문인데요.
Pattern.matches 주석에도 아래와 같은 문구가 있습니다 :)
> If a pattern is to be used multiple times, compiling it once and reusing. it will be more efficient than invoking this method each time.
아래 블로그도 참고해보세요.
https://velog.io/@edwin/JAVA-%EC%A0%95%EA%B7%9C%EC%8B%9D-%EC%84%B1%EB%8A%A5%EC%97%90-%EB%8C%80%ED%95%B4-Pattern.matcher-VS-String.matches

물론 이 미션에서는 큰 영향이 없습니다만, 이런 것이 있다 정도로 알아두시면 좋겠네요 :)